src/components/Main.css

```diff
@@ -0,0 +1,6 @@
+.App-main {
```

Comment:
Again you can leave out the App an capitalize main.

Reply:
Yep, I agree to leave it out. :P